addons/sourcemod/translations/shavit-mapchooser.phrases.txt

```diff
@@ -0,0 +1,240 @@
+"Phrases"
```

Comment:
a lot of these are in core the sourcemod translation files so we don't have to duplicate them